js/defs.bzl

```diff
@@ -41,7 +39,12 @@ def js_test(**kwargs):
         **kwargs
     )
 
-js_package = _js_package
+def js_package(**kwargs):
+    # Default the version to "0.0.0" for the public API
+    version = kwargs.pop("version", "0.0.0")
+
+    _js_package(version = version, **kwargs)
+
 JsPackageInfo = _JsPackageInfo
 
 def link_js_package(name, **kwargs):
```

Comment:
nit on docs: we should make sure these references hyperlink to each other

Reply:
also to discuss: having macros named the same as rules suggests they're not a leaky abstraction. When they leak I think we should have different names

Reply:
agreed, renamed `link_js_package` rule to `link_js_package_store` to complement `link_js_package_direct`. `link_js_package` is a composition of those two + a filegroup to expose the package directory